No hot reload on bad configuration wich block the quarkus startup and when http.port is overridden

**Describe the bug**
If you configure a bad value that block the first application start, there is no way to have hot reload.

If you start on a good value, but after plug a bbad config, the hot reload is ok. A configuration change will trigger the hot reload as wanted. 

**Expected behavior**
The hot reload is trigger if you change the configuration file, or a code change, even it never could be started correctly.

**Actual behavior**
The hot reload is broken if the app never could be started a first time.

**To Reproduce**
Steps to reproduce the behavior:
1. configure a bad jdbc url on hibernate (or flyway)
2. start the app 
3. change to good value (it won't reload)


**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Darwin 18.5.0 Darwin Kernel Version 18.5.0
 - Output of `java -version`: openjdk version "13" 2019-09-17
 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: 0.25.0

**Additional context**
(Add any other context about the problem here.)
@cescoffier the issue that we discuss after the quarkus workshop

Oh, where is maybe a more subtle thing : the jdbc url I use to reproduce is a good url but without database at the end (connexion refused because of a bad port for example).

At first startup I have theses logs : 
```
➜  rest-fight mvn compile quarkus:dev
[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for io.quarkus.workshop.super-heroes:rest-fight:jar:01
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: io.quarkus:quarkus-smallrye-health:jar -> duplicate declaration of version (?) @ line 141, column 21
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
[INFO]
[INFO] ------------< io.quarkus.workshop.super-heroes:rest-fight >-------------
[INFO] Building Quarkus Workshop :: Super-Heroes :: Fight REST API 01
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ rest-fight ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 4 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ rest-fight ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- quarkus-maven-plugin:0.25.0:dev (default-cli) @ rest-fight ---
OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.
Listening for transport dt_socket at address: 5005
19:58:19 INFO  [io.qu.de.QuarkusAugmentor] (main) Beginning quarkus augmentation
19:58:21 INFO  [io.qu.resteasy] (build-5) Resteasy running without servlet container.
19:58:21 INFO  [io.qu.resteasy] (build-5) - Add quarkus-undertow to run Resteasy within a servlet container
19:58:21 INFO  [io.qu.de.QuarkusAugmentor] (main) Quarkus augmentation completed in 1389ms

          .--.            .
     _.._ |__|  .--./)  .'|
   .' .._|.--. /.''\\  <  |             .|
   | '    |  || |  | |  | |           .' |_
 __| |__  |  | \`-' /   | | .'''-.  .'     |
|__   __| |  | /("'`    | |/.'''. \'--.  .-'
   | |    |  | \ '---.  |  /    | |   |  |
   | |    |__|  /'""'.\ | |     | |   |  |
   | |         ||     ||| |     | |   |  '.'
   | |         \'. __// | '.    | '.  |   /
   |_|          `'---'  '---'   '---' `'-'

19:58:22 WARN  [io.ne.re.dn.DefaultDnsServerAddressStreamProvider] (main) Default DNS servers: {} (Google Public DNS as a fallback)
19:58:22 WARN  [or.hi.en.jd.sp.SqlExceptionHelper] (main) SQL Error: 0, SQLState: 08001
19:58:22 ERROR [or.hi.en.jd.sp.SqlExceptionHelper] (main) Connection to localhost:5434 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
	at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
	at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits39.deploy_0(HibernateOrmProcessor$startPersistenceUnits39.zig:70)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits39.deploy(HibernateOrmProcessor$startPersistenceUnits39.zig:36)
	at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:220)
	at io.quarkus.runtime.Application.start(Application.java:94)
	at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:135)
	at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
	at io.quarkus.dev.DevModeMain.start(DevModeMain.java:94)
	at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
	at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
	at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
	at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
	at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
	... 14 more
Caused by: org.postgresql.util.PSQLException: Connection to localhost:5434 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1126)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
Caused by: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:579)
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:339)
	at java.base/java.net.Socket.connect(Socket.java:585)
	at org.postgresql.core.PGStream.<init>(PGStream.java:75)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	... 12 more
19:58:22 INFO  [io.sm.re.me.ex.MediatorManager] (main) Cancel subscriptions
19:58:22 ERROR [io.qu.de.DevModeMain] (main) Failed to start quarkus: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:284)
	at io.quarkus.runtime.Application.start(Application.java:94)
	at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:135)
	at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
	at io.quarkus.dev.DevModeMain.start(DevModeMain.java:94)
	at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
Caused by: javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
	at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
	at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits39.deploy_0(HibernateOrmProcessor$startPersistenceUnits39.zig:70)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits39.deploy(HibernateOrmProcessor$startPersistenceUnits39.zig:36)
	at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:220)
	... 5 more
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
	at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
	at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
	at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
	at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
	... 14 more
Caused by: org.postgresql.util.PSQLException: Connection to localhost:5434 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1126)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
Caused by: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:579)
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:339)
	at java.base/java.net.Socket.connect(Socket.java:585)
	at org.postgresql.core.PGStream.<init>(PGStream.java:75)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
	... 12 more

19:58:22 ERROR [io.qu.de.DevModeMain] (main) Failed to start Quarkus, attempting to start hot replacement endpoint to recover
Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: Unable to start HTTP server
	at io.quarkus.dev.DevModeMain.main(DevModeMain.java:70)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Unable to start HTTP server
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder.startServerAfterFailedStart(VertxHttpRecorder.java:122)
	at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup.handleFailedInitialStart(VertxHotReplacementSetup.java:30)
	at io.quarkus.dev.RuntimeUpdatesProcessor.startupFailed(RuntimeUpdatesProcessor.java:442)
	at io.quarkus.dev.DevModeMain.start(DevModeMain.java:101)
	at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
Caused by: java.lang.RuntimeException: Unable to start HTTP server
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder.doServerStart(VertxHttpRecorder.java:269)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder.startServerAfterFailedStart(VertxHttpRecorder.java:114)
	... 4 more
Caused by: java.util.concurrent.ExecutionException: java.net.BindException: Address already in use
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2070)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder.doServerStart(VertxHttpRecorder.java:240)
	... 5 more
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:469)
	at java.base/sun.nio.ch.Net.bind(Net.java:458)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:132)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:551)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1345)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:503)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:488)
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:984)
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:247)
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:355)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasksFrom(SingleThreadEventExecutor.java:392)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:367)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:506)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:918)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:830)
^C%
```

@Mefl 

```

 ~  Cod  qua  dem  bor  todo-jug  ./mvnw clean compile quarkus:dev
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< io.quarkus.bordeauxjug:todo-jug >-------------------
[INFO] Building todo-jug 1.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ todo-jug ---
[INFO] Deleting /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/target
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ todo-jug ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 288 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ todo-jug ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 3 source files to /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/target/classes
[INFO]
[INFO] --- quarkus-maven-plugin:0.26.1:dev (default-cli) @ todo-jug ---
Listening for transport dt_socket at address: 5005
2019-10-25 17:34:36,460 INFO  [io.qua.dep.QuarkusAugmentor] (main) Beginning quarkus augmentation
2019-10-25 17:34:37,187 INFO  [io.qua.resteasy] (build-5) Resteasy running without servlet container.
2019-10-25 17:34:37,187 INFO  [io.qua.resteasy] (build-5) - Add quarkus-undertow to run Resteasy within a servlet container
2019-10-25 17:34:37,295 INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 835ms
2019-10-25 17:34:38,480 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (main) SQL Error: 0, SQLState: 08001
2019-10-25 17:34:38,480 ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (main) Connection to localhost:5555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
        at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
        at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
        at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
        at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy_0(HibernateOrmProcessor$startPersistenceUnits25.zig:70)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy(HibernateOrmProcessor$startPersistenceUnits25.zig:36)
        at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:129)
        at io.quarkus.runtime.Application.start(Application.java:94)
        at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
        at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
        at io.quarkus.dev.DevModeMain.start(DevModeMain.java:94)
        at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
        at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
        at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
        at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
        at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
        at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
        at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
        at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
        ... 14 more
Caused by: org.postgresql.util.PSQLException: Connection to localhost:5555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
        at org.postgresql.Driver.makeConnection(Driver.java:458)
        at org.postgresql.Driver.connect(Driver.java:260)
        at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at org.postgresql.core.PGStream.<init>(PGStream.java:75)
        at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
        ... 12 more
2019-10-25 17:34:38,495 ERROR [io.qua.dev.DevModeMain] (main) Failed to start quarkus: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:193)
        at io.quarkus.runtime.Application.start(Application.java:94)
        at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
        at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
        at io.quarkus.dev.DevModeMain.start(DevModeMain.java:94)
        at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
Caused by: javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
        at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
        at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
        at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
        at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy_0(HibernateOrmProcessor$startPersistenceUnits25.zig:70)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy(HibernateOrmProcessor$startPersistenceUnits25.zig:36)
        at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:129)
        ... 5 more
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
        at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
        at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
        at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
        at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
        at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
        at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
        at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
        ... 14 more
Caused by: org.postgresql.util.PSQLException: Connection to localhost:5555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
        at org.postgresql.Driver.makeConnection(Driver.java:458)
        at org.postgresql.Driver.connect(Driver.java:260)
        at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at org.postgresql.core.PGStream.<init>(PGStream.java:75)
        at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
        ... 12 more

2019-10-25 17:34:38,496 ERROR [io.qua.dev.DevModeMain] (main) Failed to start Quarkus, attempting to start hot replacement endpoint to recover
2019-10-25 17:34:56,493 INFO  [io.qua.dev] (vert.x-worker-thread-1) File change detected: /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/src/main/resources/application.properties
2019-10-25 17:34:56,506 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-1) Beginning quarkus augmentation
2019-10-25 17:34:56,695 INFO  [io.qua.resteasy] (build-15) Resteasy running without servlet container.
2019-10-25 17:34:56,695 INFO  [io.qua.resteasy] (build-15) - Add quarkus-undertow to run Resteasy within a servlet container
2019-10-25 17:34:56,748 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-1) Quarkus augmentation completed in 242ms
Hibernate:

    drop table if exists Todo cascade
Hibernate:

    drop sequence if exists hibernate_sequence
Hibernate: create sequence hibernate_sequence start 1 increment 1
Hibernate:

    create table Todo (
       id int8 not null,
        completed boolean not null,
        ordering int4,
        title varchar(255),
        url varchar(255),
        primary key (id)
    )
Hibernate:

    alter table if exists Todo
       add constraint UK_oxfuur9jdyqshux5rm7g7ga6c unique (title)
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null)
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Hibernate with Panache', false, 1, null)
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Visit Quarkus web site', false, 2, 'https://quarkus.io')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 2', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 3', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 4', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 5', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 6', false, 3, 'https://github.com/quarkusio/quarkus/')
2019-10-25 17:34:56,995 INFO  [io.quarkus] (vert.x-worker-thread-1) Quarkus 0.26.1 started in 0.498s. Listening on: http://0.0.0.0:8080
2019-10-25 17:34:56,996 INFO  [io.quarkus] (vert.x-worker-thread-1) Profile dev activated. Live Coding activated.
2019-10-25 17:34:56,996 INFO  [io.quarkus] (vert.x-worker-thread-1) Installed features: [agroal, cdi, hibernate-orm, hibernate-validator, jdbc-postgresql, narayana-jta, resteasy, resteasy-jsonb, smallrye-openapi, swagger-ui]
2019-10-25 17:34:56,996 INFO  [io.qua.dev] (vert.x-worker-thread-1) Hot replace total time: 0.523s
Hibernate:
    select
        todo0_.id as id1_0_,
        todo0_.completed as complete2_0_,
        todo0_.ordering as ordering3_0_,
        todo0_.title as title4_0_,
        todo0_.url as url5_0_
    from
        Todo todo0_
    order by
        todo0_.ordering limit ?
2019-10-25 17:35:04,630 INFO  [io.qua.dev] (vert.x-worker-thread-3) File change detected: /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/src/main/resources/application.properties
2019-10-25 17:35:04,634 INFO  [io.quarkus] (vert.x-worker-thread-3) Quarkus stopped in 0.002s
2019-10-25 17:35:04,644 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-3) Beginning quarkus augmentation
2019-10-25 17:35:04,773 INFO  [io.qua.resteasy] (build-5) Resteasy running without servlet container.
2019-10-25 17:35:04,773 INFO  [io.qua.resteasy] (build-5) - Add quarkus-undertow to run Resteasy within a servlet container
2019-10-25 17:35:04,816 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-3) Quarkus augmentation completed in 172ms
2019-10-25 17:35:04,921 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (vert.x-worker-thread-3) SQL Error: 0, SQLState: 08001
2019-10-25 17:35:04,921 ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (vert.x-worker-thread-3) Connection to localhost:555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
        at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
        at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
        at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
        at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy_0(HibernateOrmProcessor$startPersistenceUnits89.zig:51)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy(HibernateOrmProcessor$startPersistenceUnits89.zig:70)
        at io.quarkus.runner.ApplicationImpl3.doStart(ApplicationImpl3.zig:324)
        at io.quarkus.runtime.Application.start(Application.java:94)
        at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
        at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
        at io.quarkus.dev.DevModeMain.restartApp(DevModeMain.java:195)
        at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:134)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:51)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:43)
        at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
        at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
        at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
        at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
        at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
        at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
        at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
        at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
        ... 21 more
Caused by: org.postgresql.util.PSQLException: Connection to localhost:555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
        at org.postgresql.Driver.makeConnection(Driver.java:458)
        at org.postgresql.Driver.connect(Driver.java:260)
        at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        ... 1 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at org.postgresql.core.PGStream.<init>(PGStream.java:75)
        at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
        ... 12 more
2019-10-25 17:35:04,923 ERROR [io.qua.dev.DevModeMain] (vert.x-worker-thread-3) Failed to start quarkus: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.runner.ApplicationImpl3.doStart(ApplicationImpl3.zig:388)
        at io.quarkus.runtime.Application.start(Application.java:94)
        at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
        at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
        at io.quarkus.dev.DevModeMain.restartApp(DevModeMain.java:195)
        at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:134)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:51)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:43)
        at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
        at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
        at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
        at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
        at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
        at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy_0(HibernateOrmProcessor$startPersistenceUnits89.zig:51)
        at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy(HibernateOrmProcessor$startPersistenceUnits89.zig:70)
        at io.quarkus.runner.ApplicationImpl3.doStart(ApplicationImpl3.zig:324)
        ... 12 more
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
        at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
        at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
        at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
        at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
        at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
        at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
        at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
        at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
        at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
        at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
        ... 21 more
Caused by: org.postgresql.util.PSQLException: Connection to localhost:555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
        at org.postgresql.Driver.makeConnection(Driver.java:458)
        at org.postgresql.Driver.connect(Driver.java:260)
        at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
        at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        ... 1 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at org.postgresql.core.PGStream.<init>(PGStream.java:75)
        at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
        ... 12 more

2019-10-25 17:35:04,924 INFO  [io.qua.dev] (vert.x-worker-thread-3) Hot replace total time: 0.308s
2019-10-25 17:35:17,853 INFO  [io.qua.dev] (vert.x-worker-thread-4) File change detected: /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/src/main/resources/application.properties
2019-10-25 17:35:17,863 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-4) Beginning quarkus augmentation
2019-10-25 17:35:17,985 INFO  [io.qua.resteasy] (build-3) Resteasy running without servlet container.
2019-10-25 17:35:17,986 INFO  [io.qua.resteasy] (build-3) - Add quarkus-undertow to run Resteasy within a servlet container
2019-10-25 17:35:18,036 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-4) Quarkus augmentation completed in 173ms
Hibernate:

    drop table if exists Todo cascade
Hibernate:

    drop sequence if exists hibernate_sequence
Hibernate: create sequence hibernate_sequence start 1 increment 1
Hibernate:

    create table Todo (
       id int8 not null,
        completed boolean not null,
        ordering int4,
        title varchar(255),
        url varchar(255),
        primary key (id)
    )
Hibernate:

    alter table if exists Todo
       add constraint UK_oxfuur9jdyqshux5rm7g7ga6c unique (title)
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null)
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Hibernate with Panache', false, 1, null)
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Visit Quarkus web site', false, 2, 'https://quarkus.io')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 2', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 3', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 4', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 5', false, 3, 'https://github.com/quarkusio/quarkus/')
Hibernate:
    INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 6', false, 3, 'https://github.com/quarkusio/quarkus/')
2019-10-25 17:35:18,187 INFO  [io.quarkus] (vert.x-worker-thread-4) Quarkus 0.26.1 started in 0.333s. Listening on: http://0.0.0.0:8080
2019-10-25 17:35:18,188 INFO  [io.quarkus] (vert.x-worker-thread-4) Profile dev activated. Live Coding activated.
2019-10-25 17:35:18,188 INFO  [io.quarkus] (vert.x-worker-thread-4) Installed features: [agroal, cdi, hibernate-orm, hibernate-validator, jdbc-postgresql, narayana-jta, resteasy, resteasy-jsonb, smallrye-openapi, swagger-ui]
2019-10-25 17:35:18,188 INFO  [io.qua.dev] (vert.x-worker-thread-4) Hot replace total time: 0.348s
Hibernate:
    select
        todo0_.id as id1_0_,
        todo0_.completed as complete2_0_,
        todo0_.ordering as ordering3_0_,
        todo0_.title as title4_0_,
        todo0_.url as url5_0_
    from
        Todo todo0_
    order by
        todo0_.ordering limit ?
^C2019-10-25 17:35:35,551 INFO  [io.quarkus] (Quarkus Shutdown Thread) Quarkus stopped in 0.003s
```

But as I said, the bind address is something I don't see, there is something additional in your environment

Ah i think you have an other Quarkus instance running in the background.
That would explain the issues you see. I.e always failing.

On Fri 25 Oct 2019 at 17:50, Emmanuel Bernard <notifications@github.com>
wrote:

> @Mefl <https://github.com/Mefl>
>
>
>
>  ~  Cod  qua  dem  bor  todo-jug  ./mvnw clean compile quarkus:dev
>
> [INFO] Scanning for projects...
>
> [INFO]
>
> [INFO] ------------------< io.quarkus.bordeauxjug:todo-jug >-------------------
>
> [INFO] Building todo-jug 1.0.0-SNAPSHOT
>
> [INFO] --------------------------------[ jar ]---------------------------------
>
> [INFO]
>
> [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ todo-jug ---
>
> [INFO] Deleting /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/target
>
> [INFO]
>
> [INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ todo-jug ---
>
> [INFO] Using 'UTF-8' encoding to copy filtered resources.
>
> [INFO] Copying 288 resources
>
> [INFO]
>
> [INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ todo-jug ---
>
> [INFO] Changes detected - recompiling the module!
>
> [INFO] Compiling 3 source files to /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/target/classes
>
> [INFO]
>
> [INFO] --- quarkus-maven-plugin:0.26.1:dev (default-cli) @ todo-jug ---
>
> Listening for transport dt_socket at address: 5005
>
> 2019-10-25 17:34:36,460 INFO  [io.qua.dep.QuarkusAugmentor] (main) Beginning quarkus augmentation
>
> 2019-10-25 17:34:37,187 INFO  [io.qua.resteasy] (build-5) Resteasy running without servlet container.
>
> 2019-10-25 17:34:37,187 INFO  [io.qua.resteasy] (build-5) - Add quarkus-undertow to run Resteasy within a servlet container
>
> 2019-10-25 17:34:37,295 INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 835ms
>
> 2019-10-25 17:34:38,480 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (main) SQL Error: 0, SQLState: 08001
>
> 2019-10-25 17:34:38,480 ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (main) Connection to localhost:5555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
>
> javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
>
>         at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
>
>         at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy_0(HibernateOrmProcessor$startPersistenceUnits25.zig:70)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy(HibernateOrmProcessor$startPersistenceUnits25.zig:36)
>
>         at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:129)
>
>         at io.quarkus.runtime.Application.start(Application.java:94)
>
>         at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
>
>         at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
>
>         at io.quarkus.dev.DevModeMain.start(DevModeMain.java:94)
>
>         at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
>
> Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
>
>         at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
>
>         at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
>
>         at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
>
>         at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
>
>         at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
>
>         ... 14 more
>
> Caused by: org.postgresql.util.PSQLException: Connection to localhost:5555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
>
>         at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
>
>         at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
>
>         at org.postgresql.Driver.makeConnection(Driver.java:458)
>
>         at org.postgresql.Driver.connect(Driver.java:260)
>
>         at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
>
>         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
>
>         at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
>
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
>
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>
>         at java.lang.Thread.run(Thread.java:748)
>
> Caused by: java.net.ConnectException: Connection refused (Connection refused)
>
>         at java.net.PlainSocketImpl.socketConnect(Native Method)
>
>         at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
>
>         at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
>
>         at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
>
>         at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
>
>         at java.net.Socket.connect(Socket.java:589)
>
>         at org.postgresql.core.PGStream.<init>(PGStream.java:75)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
>
>         ... 12 more
>
> 2019-10-25 17:34:38,495 ERROR [io.qua.dev.DevModeMain] (main) Failed to start quarkus: java.lang.RuntimeException: Failed to start quarkus
>
>         at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:193)
>
>         at io.quarkus.runtime.Application.start(Application.java:94)
>
>         at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
>
>         at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
>
>         at io.quarkus.dev.DevModeMain.start(DevModeMain.java:94)
>
>         at io.quarkus.dev.DevModeMain.main(DevModeMain.java:66)
>
> Caused by: javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
>
>         at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
>
>         at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy_0(HibernateOrmProcessor$startPersistenceUnits25.zig:70)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits25.deploy(HibernateOrmProcessor$startPersistenceUnits25.zig:36)
>
>         at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:129)
>
>         ... 5 more
>
> Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
>
>         at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
>
>         at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
>
>         at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
>
>         at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
>
>         at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
>
>         ... 14 more
>
> Caused by: org.postgresql.util.PSQLException: Connection to localhost:5555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
>
>         at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
>
>         at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
>
>         at org.postgresql.Driver.makeConnection(Driver.java:458)
>
>         at org.postgresql.Driver.connect(Driver.java:260)
>
>         at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
>
>         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
>
>         at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
>
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
>
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>
>         at java.lang.Thread.run(Thread.java:748)
>
> Caused by: java.net.ConnectException: Connection refused (Connection refused)
>
>         at java.net.PlainSocketImpl.socketConnect(Native Method)
>
>         at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
>
>         at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
>
>         at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
>
>         at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
>
>         at java.net.Socket.connect(Socket.java:589)
>
>         at org.postgresql.core.PGStream.<init>(PGStream.java:75)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
>
>         ... 12 more
>
>
>
> 2019-10-25 17:34:38,496 ERROR [io.qua.dev.DevModeMain] (main) Failed to start Quarkus, attempting to start hot replacement endpoint to recover
>
> 2019-10-25 17:34:56,493 INFO  [io.qua.dev] (vert.x-worker-thread-1) File change detected: /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/src/main/resources/application.properties
>
> 2019-10-25 17:34:56,506 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-1) Beginning quarkus augmentation
>
> 2019-10-25 17:34:56,695 INFO  [io.qua.resteasy] (build-15) Resteasy running without servlet container.
>
> 2019-10-25 17:34:56,695 INFO  [io.qua.resteasy] (build-15) - Add quarkus-undertow to run Resteasy within a servlet container
>
> 2019-10-25 17:34:56,748 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-1) Quarkus augmentation completed in 242ms
>
> Hibernate:
>
>
>
>     drop table if exists Todo cascade
>
> Hibernate:
>
>
>
>     drop sequence if exists hibernate_sequence
>
> Hibernate: create sequence hibernate_sequence start 1 increment 1
>
> Hibernate:
>
>
>
>     create table Todo (
>
>        id int8 not null,
>
>         completed boolean not null,
>
>         ordering int4,
>
>         title varchar(255),
>
>         url varchar(255),
>
>         primary key (id)
>
>     )
>
> Hibernate:
>
>
>
>     alter table if exists Todo
>
>        add constraint UK_oxfuur9jdyqshux5rm7g7ga6c unique (title)
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null)
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Hibernate with Panache', false, 1, null)
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Visit Quarkus web site', false, 2, 'https://quarkus.io')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 2', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 3', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 4', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 5', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 6', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> 2019-10-25 17:34:56,995 INFO  [io.quarkus] (vert.x-worker-thread-1) Quarkus 0.26.1 started in 0.498s. Listening on: http://0.0.0.0:8080
>
> 2019-10-25 17:34:56,996 INFO  [io.quarkus] (vert.x-worker-thread-1) Profile dev activated. Live Coding activated.
>
> 2019-10-25 17:34:56,996 INFO  [io.quarkus] (vert.x-worker-thread-1) Installed features: [agroal, cdi, hibernate-orm, hibernate-validator, jdbc-postgresql, narayana-jta, resteasy, resteasy-jsonb, smallrye-openapi, swagger-ui]
>
> 2019-10-25 17:34:56,996 INFO  [io.qua.dev] (vert.x-worker-thread-1) Hot replace total time: 0.523s
>
> Hibernate:
>
>     select
>
>         todo0_.id as id1_0_,
>
>         todo0_.completed as complete2_0_,
>
>         todo0_.ordering as ordering3_0_,
>
>         todo0_.title as title4_0_,
>
>         todo0_.url as url5_0_
>
>     from
>
>         Todo todo0_
>
>     order by
>
>         todo0_.ordering limit ?
>
> 2019-10-25 17:35:04,630 INFO  [io.qua.dev] (vert.x-worker-thread-3) File change detected: /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/src/main/resources/application.properties
>
> 2019-10-25 17:35:04,634 INFO  [io.quarkus] (vert.x-worker-thread-3) Quarkus stopped in 0.002s
>
> 2019-10-25 17:35:04,644 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-3) Beginning quarkus augmentation
>
> 2019-10-25 17:35:04,773 INFO  [io.qua.resteasy] (build-5) Resteasy running without servlet container.
>
> 2019-10-25 17:35:04,773 INFO  [io.qua.resteasy] (build-5) - Add quarkus-undertow to run Resteasy within a servlet container
>
> 2019-10-25 17:35:04,816 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-3) Quarkus augmentation completed in 172ms
>
> 2019-10-25 17:35:04,921 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (vert.x-worker-thread-3) SQL Error: 0, SQLState: 08001
>
> 2019-10-25 17:35:04,921 ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (vert.x-worker-thread-3) Connection to localhost:555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
>
> javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
>
>         at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
>
>         at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy_0(HibernateOrmProcessor$startPersistenceUnits89.zig:51)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy(HibernateOrmProcessor$startPersistenceUnits89.zig:70)
>
>         at io.quarkus.runner.ApplicationImpl3.doStart(ApplicationImpl3.zig:324)
>
>         at io.quarkus.runtime.Application.start(Application.java:94)
>
>         at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
>
>         at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
>
>         at io.quarkus.dev.DevModeMain.restartApp(DevModeMain.java:195)
>
>         at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:134)
>
>         at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:51)
>
>         at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:43)
>
>         at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
>
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
>
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>
>         at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
>
>         at java.lang.Thread.run(Thread.java:748)
>
> Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
>
>         at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
>
>         at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
>
>         at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
>
>         at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
>
>         at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
>
>         ... 21 more
>
> Caused by: org.postgresql.util.PSQLException: Connection to localhost:555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
>
>         at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
>
>         at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
>
>         at org.postgresql.Driver.makeConnection(Driver.java:458)
>
>         at org.postgresql.Driver.connect(Driver.java:260)
>
>         at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
>
>         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
>
>         at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
>
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
>
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>
>         ... 1 more
>
> Caused by: java.net.ConnectException: Connection refused (Connection refused)
>
>         at java.net.PlainSocketImpl.socketConnect(Native Method)
>
>         at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
>
>         at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
>
>         at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
>
>         at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
>
>         at java.net.Socket.connect(Socket.java:589)
>
>         at org.postgresql.core.PGStream.<init>(PGStream.java:75)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
>
>         ... 12 more
>
> 2019-10-25 17:35:04,923 ERROR [io.qua.dev.DevModeMain] (vert.x-worker-thread-3) Failed to start quarkus: java.lang.RuntimeException: Failed to start quarkus
>
>         at io.quarkus.runner.ApplicationImpl3.doStart(ApplicationImpl3.zig:388)
>
>         at io.quarkus.runtime.Application.start(Application.java:94)
>
>         at io.quarkus.runner.RuntimeRunner.run(RuntimeRunner.java:143)
>
>         at io.quarkus.dev.DevModeMain.doStart(DevModeMain.java:180)
>
>         at io.quarkus.dev.DevModeMain.restartApp(DevModeMain.java:195)
>
>         at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:134)
>
>         at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:51)
>
>         at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:43)
>
>         at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
>
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
>
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>
>         at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
>
>         at java.lang.Thread.run(Thread.java:748)
>
> Caused by: javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
>
>         at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
>
>         at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
>
>         at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
>
>         at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy_0(HibernateOrmProcessor$startPersistenceUnits89.zig:51)
>
>         at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits89.deploy(HibernateOrmProcessor$startPersistenceUnits89.zig:70)
>
>         at io.quarkus.runner.ApplicationImpl3.doStart(ApplicationImpl3.zig:324)
>
>         ... 12 more
>
> Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution
>
>         at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:112)
>
>         at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
>
>         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.<init>(DdlTransactionIsolatorJtaImpl.java:62)
>
>         at org.hibernate.resource.transaction.backend.jta.internal.JtaTransactionCoordinatorBuilderImpl.buildDdlTransactionIsolator(JtaTransactionCoordinatorBuilderImpl.java:46)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.getDdlTransactionIsolator(HibernateSchemaManagementTool.java:175)
>
>         at org.hibernate.tool.schema.internal.HibernateSchemaManagementTool.buildGenerationTargets(HibernateSchemaManagementTool.java:135)
>
>         at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:110)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:145)
>
>         at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:73)
>
>         at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:320)
>
>         at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
>
>         at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
>
>         ... 21 more
>
> Caused by: org.postgresql.util.PSQLException: Connection to localhost:555 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:280)
>
>         at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
>
>         at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
>
>         at org.postgresql.Driver.makeConnection(Driver.java:458)
>
>         at org.postgresql.Driver.connect(Driver.java:260)
>
>         at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:200)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:370)
>
>         at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:352)
>
>         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
>
>         at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:65)
>
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
>
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>
>         ... 1 more
>
> Caused by: java.net.ConnectException: Connection refused (Connection refused)
>
>         at java.net.PlainSocketImpl.socketConnect(Native Method)
>
>         at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
>
>         at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
>
>         at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
>
>         at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
>
>         at java.net.Socket.connect(Socket.java:589)
>
>         at org.postgresql.core.PGStream.<init>(PGStream.java:75)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:91)
>
>         at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:192)
>
>         ... 12 more
>
>
>
> 2019-10-25 17:35:04,924 INFO  [io.qua.dev] (vert.x-worker-thread-3) Hot replace total time: 0.308s
>
> 2019-10-25 17:35:17,853 INFO  [io.qua.dev] (vert.x-worker-thread-4) File change detected: /Users/emmanuel/Code/quarkus/demos/bordeauxjug/todo-jug/src/main/resources/application.properties
>
> 2019-10-25 17:35:17,863 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-4) Beginning quarkus augmentation
>
> 2019-10-25 17:35:17,985 INFO  [io.qua.resteasy] (build-3) Resteasy running without servlet container.
>
> 2019-10-25 17:35:17,986 INFO  [io.qua.resteasy] (build-3) - Add quarkus-undertow to run Resteasy within a servlet container
>
> 2019-10-25 17:35:18,036 INFO  [io.qua.dep.QuarkusAugmentor] (vert.x-worker-thread-4) Quarkus augmentation completed in 173ms
>
> Hibernate:
>
>
>
>     drop table if exists Todo cascade
>
> Hibernate:
>
>
>
>     drop sequence if exists hibernate_sequence
>
> Hibernate: create sequence hibernate_sequence start 1 increment 1
>
> Hibernate:
>
>
>
>     create table Todo (
>
>        id int8 not null,
>
>         completed boolean not null,
>
>         ordering int4,
>
>         title varchar(255),
>
>         url varchar(255),
>
>         primary key (id)
>
>     )
>
> Hibernate:
>
>
>
>     alter table if exists Todo
>
>        add constraint UK_oxfuur9jdyqshux5rm7g7ga6c unique (title)
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null)
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Hibernate with Panache', false, 1, null)
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Visit Quarkus web site', false, 2, 'https://quarkus.io')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 2', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 3', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 4', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 5', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> Hibernate:
>
>     INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project 6', false, 3, 'https://github.com/quarkusio/quarkus/')
>
> 2019-10-25 17:35:18,187 INFO  [io.quarkus] (vert.x-worker-thread-4) Quarkus 0.26.1 started in 0.333s. Listening on: http://0.0.0.0:8080
>
> 2019-10-25 17:35:18,188 INFO  [io.quarkus] (vert.x-worker-thread-4) Profile dev activated. Live Coding activated.
>
> 2019-10-25 17:35:18,188 INFO  [io.quarkus] (vert.x-worker-thread-4) Installed features: [agroal, cdi, hibernate-orm, hibernate-validator, jdbc-postgresql, narayana-jta, resteasy, resteasy-jsonb, smallrye-openapi, swagger-ui]
>
> 2019-10-25 17:35:18,188 INFO  [io.qua.dev] (vert.x-worker-thread-4) Hot replace total time: 0.348s
>
> Hibernate:
>
>     select
>
>         todo0_.id as id1_0_,
>
>         todo0_.completed as complete2_0_,
>
>         todo0_.ordering as ordering3_0_,
>
>         todo0_.title as title4_0_,
>
>         todo0_.url as url5_0_
>
>     from
>
>         Todo todo0_
>
>     order by
>
>         todo0_.ordering limit ?
>
> ^C2019-10-25 17:35:35,551 INFO  [io.quarkus] (Quarkus Shutdown Thread) Quarkus stopped in 0.003s
>
>
> But as I said, the bind address is something I don't see, there is
> something additional in your environment
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quarkusio/quarkus/issues/4815?email_source=notifications&email_token=AACJNWGYCJDB6HU3O6TDBOLQQMIULA5CNFSM4JEGKIYKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECIYQOQ#issuecomment-546408506>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACJNWCKTFFBKI6C4IQDVNTQQMIULANCNFSM4JEGKIYA>
> .
>

--- a/core/devmode/src/main/java/io/quarkus/dev/DevModeMain.java
+++ b/core/devmode/src/main/java/io/quarkus/dev/DevModeMain.java
@@ -96,10 +96,6 @@ public class DevModeMain implements Closeable {
             if (context.isAbortOnFailedStart()) {
                 throw new RuntimeException(deploymentProblem == null ? compileProblem : deploymentProblem);
             }
-            log.error("Failed to start Quarkus, attempting to start hot replacement endpoint to recover");
-            if (runtimeUpdatesProcessor != null) {
-                runtimeUpdatesProcessor.startupFailed();
-            }
         }
         Runtime.getRuntime().addShutdownHook(new Thread(new Runnable() {
             @Override
@@ -180,6 +176,20 @@ public class DevModeMain implements Closeable {
                 runner.run();
                 DevModeMain.runner = runner;
                 deploymentProblem = null;
+
+            } catch (Throwable t) {
+                deploymentProblem = t;
+                if (context.isAbortOnFailedStart() || liveReload) {
+                    log.error("Failed to start quarkus", t);
+                } else {
+                    //we need to set this here, while we still have the correct TCCL
+                    //this is so the config is still valid, and we can read HTTP config from application.properties
+                    log.error("Failed to start Quarkus, attempting to start hot replacement endpoint to recover");
+                    if (runtimeUpdatesProcessor != null) {
+                        runtimeUpdatesProcessor.startupFailed();
+                    }
+                }
+
             } finally {
                 Thread.currentThread().setContextClassLoader(old);
             }

